WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -136,6 +136,8 @@ def getCPUTime( CPUNormalizationFactor ):
   if CPUTime:
     # This is in HS06sseconds
     # We need to convert in real seconds
+    if not CPUNormalizationFactor:
+      CPUNormalizationFactor = gConfig.getValue( '/LocalSite/CPUNormalizationFactor', 0.0 )
```

Comment:
Doesn't this cause a division by 0 and ZeroDivisionError if CPUNormalizationFactor is not set in the CS